test/state/fragments/actions.test.js

```diff
@@ -174,22 +181,72 @@ describe('fragment actions', () => {
 
     describe('test fetchWidgetTypes', () => {
       it('action payload contains widgetTypes list', () => {
+        getWidgetTypes.mockReturnValue(new Promise(resolve => resolve(WIDGET_TYPES_PAYLOAD)));
```

Comment:
Please use `import { mockApi } from 'test/testUtils';` to have a realistic response, and remove `payload` from mocks.

Reply:
i don't use mockApi because this api is not already migrate using mockRequest